flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/calcite/FlinkPlannerImpl.scala

```diff
@@ -156,6 +156,10 @@ class FlinkPlannerImpl(
           }
           richExplain.setOperand(0, validatedStatement)
           richExplain
+        // Insert requires validation but no row validation
```

Comment:
@Airblader unfortunately I had to resort to this code, because the `validator.validate(sqlNode)` will have the side effect of performing additional validation on the rows (which breaks several tests). Does it makes sense?

Reply:
I think this is OK, but we should link this comment to the same issue we create for the topic of properly running INSERT validation.